### PR TITLE
 [pack] Update integration tests build

### PR DESCRIPTION
**Pull request by Francisco-Gamino**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
    * [ ] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: build/update-hostreferences.ps1 [hunk 1]
```diff
@@ -38,6 +38,18 @@ if ($packagesToUpdate.Count -eq 0)
     WriteLog "There are no packages to update in '$url'" -Throw
 }
 
+<#
+TODO: Need to create a integrationTestsBuildManifest.json
+
```
Comment (pragnagopa) [inline build/update-hostreferences.ps1:117]: please remove TODO for now

**Review by pragnagopa — COMMENTED** [review]
LGTM! @yojagad  - please take look as well

File: build/upload-functionsZipToStorageAccount.ps1 [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+WriteLog -Message "Script started."
+
+if (-not (Test-Path $SourcePath))
+{
+    throw "SourcePath '$SourcePath' does not exist."
+}
+
+$CONTAINER_NAME = "functionsbuilds"
+$FUNC_RUNTIME_VERSION = '3'
```
Comment (pragnagopa) [inline build/upload-functionsZipToStorageAccount.ps1:104]: this should input param ? as we need to generate and upload nightly builds for V4 as well?

**Comment by liliankasem** [comment]
> LGTM! @yojagad - please take look as well

Poke @yojagad to give this another look, would love to close stale PRs :)

**Comment by fabiocav** [comment]
Closing PR as action here is required. If this is still relevant, please address conflicts and reopen. Thanks!

**Pull request closed without merging**